schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL(12,2)
+    weight_kg DECIMAL(12,2),
+    species VARCHAR(200)
 );
```

Comment:
- You have done great so far, :+1:  But there are few changes you need to make before moving to the next milestone. :+1: 

- [ ] Editing already ran migration is not a recommended way to add new fields actually, you should use this statement 
```sh
ALTER TABLE animals ADD species varchar(255); 
```
instead!

Reply:
- Refactored to include `ALTER TABLE animals ADD species VARCHAR(200); ` statement to resolve bug.